### DISCUSSION TITLE
Modify RGB contrast and do a Vector transform from world to camera?

### DISCUSSION THREAD

**Discussion by Codesterz** [bnpr/Malt#192] [Q&A] [answered]
As the topic title says there are 2 things I'm having trouble figuring out how to do with nodes. The first is how to set up nodes to adjust the contrast of a procedural noise. The second thing is shown in a screen shot from eevee. Any help would be appreciated honestly not sure how to make them happen. 
![Node02](https://user-images.githubusercontent.com/52447070/145518440-0ca890fc-1f24-4e0b-b697-535ea39c6468.jpg)

**Comment by pragma37** [comment, marked as answer]
You can use the transform_point, transform_vector and transform_normal nodes:
![imagen](https://user-images.githubusercontent.com/36610999/145612641-10d05e93-1983-45fe-9788-579664fadd13.png)

There's no built-in contrast function (yet?), but if you google **contrast glsl** you can find plenty of examples:
https://alaingalvan.tumblr.com/post/79864187609/glsl-color-correction-shaders

You can copy-paste any of those functions into your shader library and they'll be available as nodes.
